src/legacy/ui/public/url/url.js

```diff
@@ -121,12 +119,7 @@ export function KbnUrlProvider($injector, $location, $rootScope, $parse, Private
           }));
       }
 
-      // append uriescape filter if not included
-      if (expr.indexOf('uriescape') === -1) {
-        expr += '|uriescape';
-      }
-
-      return $parse(expr)(paramObj);
```

Comment:
@eliperelman you may be interested in changes

Reply:
@eliperelman would be glad to chat about it